src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,46 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size()로 Set의 크기 확인")
+    void sizeTest(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("contains()로 set에 있는 값 확인")
+    void containsTest(int number){
```

Comment:
```suggestion
    @ValueSource(ints = {1, 2, 3})
    @DisplayName("contains()로 set에 있는 값 확인")
    void containsTest(int number) {
```
공백이 들어가는 것이 컨벤션인 점 참고부탁드려요 :)